File: java-junit5-no-plugin-old/src/integration/java/com/coditory/sandbox/ClasspathSetupIntegrationSpec.java
package com.coditory.sandbox;

import org.junit.jupiter.api.Test;

import static com.coditory.sandbox.base.ClasspathFileReader.readFile;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class ClasspathSetupIntegrationSpec {
    @Test
    public void shouldReadATxtFileFromMain() {
        assertEquals("main-a", readFile("a.txt"));
    }

    @Test
    public void shouldReadBTxtFileFromTest() {
        assertEquals("test-b", readFile("b.txt"));
    }

    @Test
    public void shouldReadCTxtFileFromIntegration() {
        assertEquals("integration-c", readFile("c.txt"));
    }
}

File: kotlin-junit5-no-plugin/src/integrationTest/kotlin/com/coditory/sandbox/ClasspathSetupIntegrationSpec.kt
package com.coditory.sandbox

import com.coditory.quark.context.Context
import com.coditory.sandbox.base.ClasspathFileReader.readFile
import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class ClasspathSetupIntegrationSpec {
    @Test
    fun shouldReadATxtFileFromMain() {
        assertEquals("main-a", readFile("a.txt"))
    }

    @Test
    fun shouldReadBTxtFileFromTest() {
        assertEquals("test-b", readFile("b.txt"))
    }

    @Test
    fun shouldReadCTxtFileFromIntegration() {
        assertEquals("integration-c", readFile("c.txt"))
    }
}
